apps/api/src/domain/models/photograph/photograph.commands.module.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandModule } from '@coscrad/commands';
+import { Module } from '@nestjs/common';
+import { CreatePhotograph, CreatePhotographCommandHandler } from '.';
+import { IdGenerationModule } from '../../../lib/id-generation/id-generation.module';
+import { PersistenceModule } from '../../../persistence/persistence.module';
+
+@Module({
+    imports: [PersistenceModule, CommandModule, IdGenerationModule],
+    providers: [
+        CreatePhotographCommandHandler,
+        ...[CreatePhotograph].map((Ctor) => ({
+            provide: Ctor,
+            useValue: Ctor,
+        })),
+    ],
+    exports: [CreatePhotograph],
+})
+export class PhotographCommandsModule {}
```

Comment:
Oh this is interesting. I suppose I probably did this for one of the other resources. I think I am aiming at being able to bring smaller modules into tests.

Reply:
That said let's put some thought to this pattern.

Reply:
Yeah, I was following the Term code rather than Vocabulary List so added this.

Reply:
I'd say if we don't have a use case for it, let's not do this right now. But if there is a reason to do this, let's go there. I'm guessing it has to do with avoiding `createTestModule`, but at the same time, the way I set up query tests has changed significantly already and it may just be a vestige.